scenarios/17_Sneaking_out_of_Raedwood.cfg

```diff
@@ -571,6 +582,17 @@
         [/objectives]
     [/event]
 
+    [event]
+        name=recall
+        [filter]
+            # wmllint: recognize Karl Regven
+            id=Karl Regven
+        [/filter]
+        {MSGm_Gawen _"Wait... Karl... why aren't you guarding Karen?"}
+        {MESSAGE (Karl Regven) (portraits/regven.png) (Karl Regven) _"You asked me to come with you, my lord, did you not?"}
+        {MSGm_Gawen _"Right, that I did... I'm sure she'll be fine..."}
```

Comment:
And, "Right, I did... I'm sure she'll be fine..."

Reply:
changed in [efba779](https://github.com/nemaara/A_New_Order/pull/43/commits/efba77972af8c9bcc1fc309bcc70c2a6748d2016)